.github/workflows/main.yml

```diff
@@ -103,7 +103,9 @@ jobs:
       - name: Build docs
         if: matrix.task == 'docs'
         run: |
-          poetry run rtai
+          [[ ! -d docs/_build/html ]]
```

Comment:
how does one recover if the html directory is present?

Reply:
wouldn't it be better to "force" (don't fail if already not there) a removal of the html directory?

Reply:
It's a gitignored file, so within the CI it shouldn't be there.

I would be concerned if it was there and would probably prefer a noisy failure as opposed to it silently fixing the issue ...